### ISSUE TITLE
Subscription gateway submission failing

### ISSUE THREAD

**Issue by bpurtoncc** [digital-pros/commerce-authorize#39]
When I am using the default form I get the following error when submitting the form:

> Unable to start the subscription. Something went wrong while scheduling the recurring payment in Authorize.net (Payment information is required.) Please try again.

Plugin version: 
4.0.6
--
Commerce version:
4.3.2
-- 

My form looks like:

```
<form method="post" action="" id="paymentForm" name="subscriptionForm">
  {{ csrfInput() }}
  {{ actionInput('commerce/subscriptions/subscribe') }}
  {{ redirectInput(settings.redirectUrl) }}
  {{ successMessageInput(settings.successMessage) }}
  {{ hiddenInput('planUid', settings.plan.uid|hash) }} 
  {% namespace settings.plan.gateway.handle|commercePaymentFormNamespace %}
    {{ settings.plan.gateway.getPaymentFormHtml({})|raw }}
  {% endnamespace %}
</form>
```

**Comment by benlobach** [comment]
@bpurtoncc - Great to hear from you! It appears this is due to a bug in Craft Commerce core, and we're waiting on a fix from the Craft Commerce team.

https://github.com/craftcms/commerce/blob/b2756caabcc3b5e39b95288a7a8bd5e8bcb1d339/src/controllers/SubscriptionsController.php#L198 

Adding an exclamation mark before the isset on this line corrects the issue where a payment source is required before the subscription is setup.

**Comment by bpurtoncc** [comment]
@benlobach Do you know if there was an issue submitted for this in the commerce repo?

**Comment by benlobach** [comment]
I don't believe there was, but the support team has confirmed that it has been fixed, and will be included in the next release.

**Comment by benlobach** [comment]
@bpurtoncc - This should now be fixed in Commerce 4.4.1.1.
